compose/backend.yml

```diff
@@ -126,23 +126,20 @@ services:
       TESTING_SUPPORT_ENABLED: "true"
 
   fr-am:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-fr-am:1.0.0
-    environment:
-      JAVA_OPTS: -Xms128m -Xmx512m
```

Comment:
same as above keep these

Reply:
Anand what's the removal of JAVA_OPTS please?

Reply:
@mario-paniccia  Not sure who made those changes, as I mentioned above it wasn't from me.